src/menu.tsx

```diff
@@ -91,38 +92,49 @@ export function get_menu_items(
             }
         );
     } else if (selected.length === 1) {
+        const item = selected[0];
+
+        if (item.type === 'reg' && item.size && item.size < 1000000) // 1MB
```

Comment:
Bikeshedding: This simple editor isn't really suitable for such large files, especially if they are binary. This feels more like in the "kB" magnitude.

Reply:
Agreed, what do we pick?

Reply:
I'd say limit to 200k, @garrett @martinpitt what do you think?

Reply:
That seems sensible to me. :+1: 

( :thought_balloon: I wonder what size starts causing performance problems? I doubt anywhere near that; probably several megabytes.)